### PR TITLE
[Relay] Improved `multiply_rewrite` for gluoncv_ssd quantization

### PR DESCRIPTION
**Pull request by liangfu**
@ZihengJiang @vinx13 please review.

### CODE REVIEW

File: python/tvm/relay/quantize/_annotate.py [hunk 1]
```diff
@@ -166,7 +166,7 @@ def multiply_rewrite(ref_call, new_args, ctx):
 
     if lhs_kind is None and rhs_kind is None:
         return None
-    if lhs_kind == QAnnotateKind.ACTIVATION and rhs_kind is None:
+    if lhs_kind in [QAnnotateKind.ACTIVATION, QAnnotateKind.INPUT] and rhs_kind is None:
```
Comment (vinx13) [inline python/tvm/relay/quantize/_annotate.py]: add a check to `attach_simulated_quantize` below to prevent quantizing lhs to INPUT if lhs_kind is INPUT

Comment (liangfu) [reply python/tvm/relay/quantize/_annotate.py]: I'm not quite familiar with this, but what shall `lhs_expr` be if `lhs_kind` is INPUT?

Comment (vinx13) [reply python/tvm/relay/quantize/_annotate.py]: it is the same from input, `new_args[0]`

Comment (liangfu) [reply python/tvm/relay/quantize/_annotate.py]: okay, i will revise accordingly.

**Pull request merged**